### ISSUE TITLE
Bug when Training Models

### ISSUE THREAD

**Issue by ZhifengWu7** [amazon-science/tabsyn#21]
When trying to train a model using the following command:
python main.py --dataname adult --method smote --mode train

I encountered the following error:
Module baselines.smote.main not found.

Does anyone know how to fix this bug?

**Comment by QDKStorm** [comment]
> When trying to train a model using the following command: python main.py --dataname adult --method smote --mode train
> 
> I encountered the following error: Module baselines.smote.main not found.
> 
> Does anyone know how to fix this bug?

I met the same problem when I run this command too
`python main.py --dataname adult --method stasy --mode train`
did you fix that?

**Comment by ZhifengWu7** [comment]
YES, the reason is that I did not install the scikit-learn package successfully.

**Comment by hengruizhang98** [comment]
Sorry for the late reply, I cannot receive the notification emails since I am not the owner of this repo. 

The above issues seem to be caused by the inconsistent package versions, e.g., scikit-learn. I will update this repo to specify the package versions.

**Comment by ZhifengWu7** [comment]
Thank for you reply. I have fixed the bug, and answered another person who
met the same issue in your repo.

**Comment by zyf343865604** [comment]
> YES, the reason is that I did not install the scikit-learn package successfully.

I install the scikit-learn package successfully，why i met the same issue

**Comment by zyf343865604** [comment]
> > 尝试使用以下命令训练模型时： python main.py --dataname adult --method smote --mode train
> > 我遇到了以下错误：找不到模块 baselines.smote.main。
> > 有谁知道如何修复这个错误？
> 
> 当我运行这个命令时，我也遇到了同样的问题，你解决了吗？`python main.py --dataname adult --method stasy --mode train`
did you solve

**Comment by QDKStorm** [comment]
> > > 尝试使用以下命令训练模型时： python main.py --dataname adult --method smote --mode train
> > > 我遇到了以下错误：找不到模块 baselines.smote.main。
> > > 有谁知道如何修复这个错误？
> > 
> > 
> > 当我运行这个命令时，我也遇到了同样的问题，你解决了吗？`python main.py --dataname adult --method stasy --mode train`
> > did you solve

The inconsistent scikit-learn version might be the cause. I fixed it by resetting my scikit-learn version.
